eng/native/configuretools.cmake

```diff
@@ -39,19 +39,21 @@ if(NOT WIN32)
     set(${var} ${EXEC_LOCATION_${exec}} PARENT_SCOPE)
   endfunction()
 
-  locate_toolchain_exec(ar CMAKE_AR)
-  locate_toolchain_exec(link CMAKE_LINKER)
-  locate_toolchain_exec(nm CMAKE_NM)
+  if(NOT CLR_CMAKE_TARGET_ANDROID OR CROSS_ROOTFS)
```

Comment:
Why was this change required when we have 100% compiling master? I have not found NDK without these tools.

Reply:
I'll check, might just be a leftover from my branch.